rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java

```diff
@@ -89,7 +89,7 @@ public static void setLogLevel(@RxBleLog.LogLevel int logLevel) {
      * Returns an infinite observable emitting BLE scan results.
      * Scan is automatically started and stopped based on the Observable lifecycle.
      * Scan is started on subscribe and stopped on unsubscribe. You can safely subscribe multiple observers to this observable.
```

Comment:
Maybe `Subscribe starts scan when unsubsribe stops it`

Reply:
`Scan is started when the Observable is subscribed and stopped when unsubscribed.` I prefer like this.